case-assignment/person-search.vbs

```diff
@@ -0,0 +1,247 @@
+	IF run_locally = FALSE or run_locally = "" THEN	   'If the scripts are set to run locally, it skips this and uses an FSO below.
+		IF use_master_branch = TRUE THEN			   'If the default_directory is C:\DHS-MAXIS-Scripts\Script Files, you're probably a scriptwriter and should use the master branch.
+			FuncLib_URL = "https://raw.githubusercontent.com/Hennepin-County/MAXIS-scripts/master/MASTER%20FUNCTIONS%20LIBRARY.vbs"
+		Else											'Everyone else should use the release branch.
+			FuncLib_URL = "https://raw.githubusercontent.com/Hennepin-County/MAXIS-scripts/master/MASTER%20FUNCTIONS%20LIBRARY.vbs"
+		End if
+		SET req = CreateObject("Msxml2.XMLHttp.6.0")				'Creates an object to get a FuncLib_URL
+		req.open "GET", FuncLib_URL, FALSE							'Attempts to open the FuncLib_URL
+		req.send													'Sends request
+		IF req.Status = 200 THEN									'200 means great success
+			Set fso = CreateObject("Scripting.FileSystemObject")	'Creates an FSO
+			Execute req.responseText								'Executes the script code
+		ELSE														'Error message
+			critical_error_msgbox = MsgBox ("Something has gone wrong. The Functions Library code stored on GitHub was not able to be reached." & vbNewLine & vbNewLine &_
+                                            "FuncLib URL: " & FuncLib_URL & vbNewLine & vbNewLine &_
+                                            "The script has stopped. Please check your Internet connection. Consult a scripts administrator with any questions.", _
+                                            vbOKonly + vbCritical, "BlueZone Scripts Critical Error")
+            StopScript
+		END IF
+	ELSE
+		FuncLib_URL = "C:\MAXIS-scripts\MASTER FUNCTIONS LIBRARY.vbs"
+		Set run_another_script_fso = CreateObject("Scripting.FileSystemObject")
+		Set fso_command = run_another_script_fso.OpenTextFile(FuncLib_URL)
+		text_from_the_other_script = fso_command.ReadAll
+		fso_command.Close
+		Execute text_from_the_other_script
+	END IF
+END IF
+'END FUNCTIONS LIBRARY BLOCK================================================================================================
+'CHANGELOG BLOCK ===========================================================================================================
+'
+'
+'END CHANGELOG BLOCK =======================================================================================================
+Dialog1 = ""												'updated the dialog name because the dialog functionality requires this
+BeginDialog Dialog1, 0, 0, 146, 80, "Select Search Type"
+  DropListBox 10, 14, 110, 5, "Select"+chr(9)+"SSN"+chr(9)+"Name and DOB"+chr(9)+"Last Name and First Name", Search_type
+  ButtonGroup ButtonPressed
+    OkButton 10, 50, 45, 20
+  ButtonGroup ButtonPressed
+    CancelButton 60, 50, 45, 20
+EndDialog
+
+
+DO
+			err_msg = ""
+			Dialog Dialog1
+			If ButtonPressed = 0 then Stopscript
+			If Search_type = "Select" THEN err_msg = err_msg & vbCr & "Please select SSN or Name and DOB."
```

Comment:
Should this error handling also include verbiage for Last Name and First Name since this is a valid option from your dropdown?

Reply:
This is part of the script from MNIT that I am letting run as is. I would make a lot of changes here if it were up to me.

Reply:
Touche